openapi/p4pa-auth.openapi.yaml

```
@@ -602,6 +602,35 @@ paths:
           description: ToS acceptance missing
         '429':
           description: Too Many Requests
+  /am/operators/{organizationIpaCode}/byExternalUserId:
+    delete:
+      tags:
+        - authz
+      operationId: deleteOrganizationOperatorByExternalUserId
+      parameters:
+        - name: organizationIpaCode
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: externalUserId
```

custom header name should start with X-
```suggestion
        - name: X-externalUserId
```
let's also add a description to explain that it's the iam userId